README.md

```diff
@@ -92,14 +126,52 @@ TF_LOG_PROVIDER=INFO terraform refresh
 TF_LOG_PROVIDER=INFO terraform show
 ```
 
-### Stop Tasks
+### Stop Task
 
 ```
 TF_LOG_PROVIDER=INFO terraform destroy
 ```
 
 This terminates the `machine` (if still running), downloads `output`, and removes the persistent `disk_size` storage.
 
+## How it Works
+
+This diagram may help to see what TPI does under-the-hood:
+
+```mermaid
+flowchart LR
+subgraph tpi [what TPI manages]
+direction LR
+    subgraph you [what you manage]
+        direction LR
+        A([Personal Computer])
+    end
+    B[("Cloud Storage (low cost)")]
+    C{{"Cloud instance scaler (zero cost)"}}
+    D[["Cloud (spot) Instance"]]
+    A ---> |create cloud storage| B
+    A --> |create cloud instance scaler| C
+    A ==> |upload script & workdir| B
+    A -.-> |"offline (lunch break)"| A
+    C -.-> |"(re)provision instance"| D
+    D ==> |run script| D
+    B <-.-> |persistent workdir cache| D
+    D ==> |script end,\nshutdown instance| B
+    D -.-> |outage| C
+    B ==> |download output| A
```

Comment:
```suggestion
    A ---> |2. create cloud storage| B
    A --> |1. create cloud instance scaler| C
    A ==> |3. upload script & workdir| B
    A -.-> |"4. offline (lunch break)"| A
    C -.-> |"5. (re)provision instance"| D
    D ==> |7. run script| D
    B <-.-> |6. persistent workdir cache| D
    D ==> |8. script end,\nshutdown instance| B
    D -.-> |outage| C
    B ==> |9. download output| A
```

Reply:
Looks good.

Reply:
Cool diagram but it's not readable without clicking the `<->` button (not necessarily obvious usability). Any chance to make it taller and less long i.e. fold the diagram somehow?